providers/amazon/README.rst

```diff
@@ -104,5 +103,25 @@ Dependent package
 `apache-airflow-providers-ssh <https://airflow.apache.org/docs/apache-airflow-providers-ssh>`_                            ``ssh``
 ========================================================================================================================  ====================
 
+Optional dependencies
```

Comment:
Very cool - can you please turn it into a table (same as dependencies above ?) this reads way better.

Also HTML escaping should be removed and the versions should be surrounded with `` to make it "code block" and not spell-checked.